packages/react/src/card/card.tsx

```diff
@@ -42,7 +42,7 @@ export const CardBody = forwardRef<HTMLDivElement, CardBaseProps>(
     const Component = asChild ? Slot : "div";
     return (
       <Component {...rest} className={clsx("hds-card__body", className as undefined)} ref={ref}>
-        {children}
+        <div>{children}</div>
```

Comment:
❓ Why do we need to wrap it in a `div`

Reply:
Doesn't have to be a div, but it needs a layer for vertical centering at a level and approach that doesn't mess up the other behavior at different widths.

Reply:
But I see it could be clearer with a named wrapper instead. I'll do that.